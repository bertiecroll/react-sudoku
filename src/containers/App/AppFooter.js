import React from 'react'

export function AppFooter () {
  return (
    <footer className="AppFooter">
      <a>Create React App</a>
      <a>Redux</a>
      <a>React Router</a>
    </footer>
  )
}
