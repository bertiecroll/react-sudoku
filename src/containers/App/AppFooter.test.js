import React from 'react';
import { shallow } from 'enzyme'
import { AppFooter } from './AppFooter';

describe('AppFooter', () => {
  it('renders component content', () => {
    const wrapper = shallow(<AppFooter />)
    expect(wrapper).toMatchSnapshot()
  })
})
