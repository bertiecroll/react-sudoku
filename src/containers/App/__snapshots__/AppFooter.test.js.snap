// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppFooter renders component content 1`] = `
<footer
  className="AppFooter"
>
  <a>
    Create React App
  </a>
  <a>
    Redux
  </a>
  <a>
    React Router
  </a>
</footer>
`;
